src/customer-service/Actions/CreateIncidentAction.cs

```diff
@@ -0,0 +1,217 @@
+﻿using customer_service.Factories;
+using customer_service.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace customer_service.Actions
+{
+    public class CreateIncidentAction
+    {
+        /**
+         * Class: CreateIncidentAction
+         * Purpose: Prompts the user for information about a new incident and saves that incident to the database
+         * Author: Garrett Vangilder
+         * Methods:
+         *     static void ReadInput() - Prompts for incident info and saves to DB
+         */
+        public CreateIncidentAction()
+        {
+
+        }
+        /**
+        * Purpose: Reads user input for new incident information
+        * Return:
+        *     void
+        */
+        public static void ReadInput()
+        {
+            Console.WriteLine(@"
+                ====================================
+                BANGAZON INC CUSTOMER SERVICE PORTAL
+                ====================================");
+            Console.WriteLine("Enter the customers first and last name to start:");
+            Console.WriteLine("***********************************************************************");
+
+            CustomerFactory customerFactory = new CustomerFactory();
+            List<Customer> customerList = customerFactory.getAll();
+            foreach (Customer customer in customerList)
+            {
+                Console.WriteLine($"{customer.CustomerId} { customer.FirstName} {customer.LastName} ");

```

Comment:
How about we leave off `CustomerId` from this WriteLine to eliminate confusion for the user